internal/web/commands.go

```diff
@@ -0,0 +1,76 @@
+package web
+
+import (
+	"fmt"
+	"io"
+	"os/exec"
+)
+
+func buildRawDisk(containerImage, outputDir string) string {
```

Comment:
cant this hook into the actual methods if aurora directly? I guess we lose the output that way?

Reply:
Im worried that if we change something in the inputs we wont notice until this is broken :(

Reply:
good point, created a ticket https://github.com/kairos-io/kairos/issues/3268